openmc/mgxs_library.py

```diff
@@ -856,9 +857,35 @@ def set_scatter_mgxs(self, scatter, nuclide='total', xs_type='macro'):
         check_value('domain_type', scatter.domain_type,
                     ['universe', 'cell', 'material'])
 
+        # Methods of representing anisotropic scattering besides
+        # Legendre expansions have not been implemented yet in openmc.mgxs.
+        # Therefore check to make sure the XsData has been set to
+        # legendre scattering.
+        if (self.scatt_type != 'legendre'):
```

Comment:
I agree it is more verbose and useful, but since this change would also reflect a change in the mgxs.xml structure too (to be consistent), do you mind if I hold off on making this change until we get to the HDF5 library format replacing the XML?  That way I'll already have to update all my models, benchmarks, examples, notebooks, etc, anyways.  So the breakage of compatibility doesn't matter